viral/ensemble_reactivation.py

```
@@ -1259,6 +1266,78 @@ def all_mice_ensemble_results_plots(
+
+    # Align signs
+    for i in range(W_py.shape[1]):
+        if np.dot(W_py[:, i], W_mat[:, i]) < 0:
+            W_py[:, i] *= -1
+
+    test_subspace(W_py, W_mat)
+    # assert np.allclose(W_py, W_mat, atol=1e-3)
+    print("All close")
+
+
+def test_subspace(W1: np.ndarray, W2: np.ndarray) -> None:
```

Ruff (0.13.3)

Redefinition of unused `test_subspace` from line 1132

(F811)